docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,187 @@
+# Getting started with supercomputing at CSC
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computing! This page provides guidance for getting started with
+using our HPC resources.
+
+It is recommended that new users complete the CSC Computing Environment course,
+which provides an in-depth introduction to CSC services. The next teaching
+instance can be found in the
+[CSC training calendar](https://csc.fi/en/trainings/training-calendar/).
+A [self-learning implementation](https://csc.fi/en/training-calendar/csc-computing-environment-self-learning/)
+is also available. The
+[practical exercises](https://csc-training.github.io/csc-env-eff/) are
+accessible without having to sign up, and quite useful by themselves.
+
+## Available systems
+
+!!! note "Terms of use"
+    Note that in order to use any of the systems, you must first accept its
+    terms of use in the MyCSC service.
```

Comment:
This is in most cases not needed for Puhti, Mahti and Allas anymore. Remove to avoid confusion.
```suggestion
```